packages/tocco-util/src/actions/utils/report.js

```diff
@@ -0,0 +1,115 @@
+import PropTypes from 'prop-types'
+
+const GROUP_GENERAL = 'generalSettings'
+const GROUP_RECIPIENT = 'recipientSettings'
+
+export const getGroupedValues = (settingsDefinition, values) => {
+  const groupValues = name => {
+    return {
```

Comment:
superfluous return. line 10 again.

but that's fine with me - I guess readability is better this way?